src/AstriaBridgeableERC20.sol

```diff
@@ -39,10 +42,13 @@ contract AstriaBridgeableERC20 is IAstriaWithdrawer, ERC20 {
         BASE_CHAIN_ASSET_DENOMINATION = _baseChainAssetDenomination;
         DIVISOR = 10 ** (decimals - _baseChainAssetPrecision);
         BRIDGE = _bridge;
+        WITHDRAWAL_FEE = _withdrawalFee;
+        FEE_RECIPIENT = _feeRecipient;
     }
 
     modifier sufficientValue(uint256 amount) {
         require(amount / DIVISOR > 0, "AstriaBridgeableERC20: insufficient value, must be greater than 10 ** (TOKEN_DECIMALS - BASE_CHAIN_ASSET_PRECISION)");
+        require(msg.value >= WITHDRAWAL_FEE, "AstriaBridgeableERC20: insufficient withdrawal fee");
```

Comment:
shouldn't the withdrawal fee be different for withdrawing to sequencer vs ibc chain? since this main purpose is to cover ibc relayer costs, the fees should be different

Reply:
open question for @joroshiba